docs/quickstart.md

```diff
@@ -37,7 +37,10 @@ The scripts from this quickstart guide set up a local environment consisting of
 two Trino servers and a PostgreSQL database running in Docker, and a Trino
 Gateway server running in the host operating system. If you are using a Mac and
 have the `brew` package manager installed, this script attempts to install
-`psql`, if not you must install `psql` through other means.
+`psql`, if not you must install `psql` through other means. 
+
+Use the following command in your shell startup script or manually:
+- `export PATH="/opt/homebrew/opt/libpq/bin:$PATH"`
```

Comment:
This section should be rewritten to use psql in a container instead of local psql. Suggesting brew install doesn't make sense as it already exists in the container. 

Also, we already have TrinoGatewayRunner that initialize the database automatically. I feel this section is increasing managing costs with less benefits.

Reply:
I think this section is for users to play with gateway without any mvn builds, just use public gateway image.


> This section should be rewritten to use psql in a container instead of local psql. Suggesting brew install doesn't make sense as it already exists in the container.

Now that you point out, we can change this by running psql within container. What do you think?
```bash
# before
psql -U postgres -h localhost -c 'CREATE DATABASE gateway'
psql  -U postgres -h localhost -d gateway -f gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql
# after
docker exec local-postgres psql -U postgres -h localhost -c 'CREATE DATABASE gateway'
docker cp gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql local-postgres:/tmp
docker exec local-postgres psql -U postgres -h localhost -d gateway -f /tmp/gateway-ha-persistence-postgres.sql

```

Reply:
@willmostly any thoughts?

Reply:
> I think this section is for users to play with gateway without any mvn builds, just use public gateway image with postgre image

Mentioning docker compose yml of `docker` directory should be sufficient. We should avoid managing duplicated information.

Reply:
This predated the gateway docker image, so its worthwhile to consider if a play environment that uses the bare jars is still worthwhile to maintain. However as @mosabua says below we should evaluate this as part of a strategic reorganization.

Reply:
This command downloads jar from maven central. It means no-difference from using docker container. 
I still disagree with having this commands in quickstart.

Reply:
We have to pull this apart for quick start with docker and quick start with jar as soon as possible. However that does not necessarily have to be part of this PR.